app/views/simulator/edit.html.erb

```diff
@@ -817,6 +818,30 @@
     </div>
   </div>
 </div>
+<script>
+function toggleCaret() {
+    console.log("clicking");
+
+    var tabsBar = $('#tabsBar');
+    var tabsCaret = $('#tabsCaret');
+
+    if (tabsCaret.hasClass('fa-solid') && tabsCaret.hasClass('fa-square-caret-down')) {
+        tabsCaret.removeClass('fa-solid fa-square-caret-down');
+        tabsCaret.addClass('fa-solid fa-square-caret-up');
+        tabsBar.css({
+            'height': '27px',
+            'overflow': 'hidden'
+        });
+    } else if (tabsCaret.hasClass('fa-solid') && tabsCaret.hasClass('fa-square-caret-up')) {
+        tabsCaret.removeClass('fa-solid fa-square-caret-up');
+        tabsCaret.addClass('fa-solid fa-square-caret-down');
+        tabsBar.css({
+            'height': 'auto',
+            'overflow': 'visible'
+        });
+    }
+}
```

Comment:
It will be better to write this codes in vanilla js instead of jquery

Reply:
ok, I will do that but may I know the reason particularly not using jquery?!